.travis.yml

```diff
@@ -19,7 +19,6 @@ install:
 - make tools
 
 script:
-- make lint
```

Comment:
This will make Travis happy 😄 but won't make all these lint issues go away...

Reply:
@dannyk81  currently its failing for gofmt, I did run go fmt on these two files not sure why its failing again on those files, due to this its not building the images, I ran all tests passed
https://travis-ci.org/terraform-providers/terraform-provider-bigip/builds/475056797?utm_source=github_status&utm_medium=notification